packages/dd-trace/test/appsec/iast/analyzers/resources/mquery-vulnerable-method.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const mquery = require('mquery')
```

Comment:
If you are requiring `mquery` here, you can't be sure that you are using the right version that you have loaded in the test.
You should pass `mquery` object as method parameter instead of require it here.

Reply:
true